src/raftstore/store/worker/pd.rs

```diff
@@ -200,7 +307,14 @@ impl<T: PdClient> Runner<T> {
         router: RaftRouter,
         db: Arc<DB>,
         scheduler: Scheduler<Task>,
+        store_heartbeat_interval: u64,
     ) -> Runner<T> {
+        let interval = Duration::from_secs(store_heartbeat_interval).div_f64(2.0);
```

Comment:
There is `impl Div<u32> for Duration` in std lib. BTW make `2.0` as a constant is better.